### PR TITLE
Fix sdist and stack build by moving LICENSE (and others) into subdirs

### PR DESCRIPTION
**Pull request by hasufell**
This does not split README/CHANGELOG properly, but just duplicates them,
so the build doesn't fail anymore.

Also see https://github.com/commercialhaskell/stack/issues/2500#issuecomment-585760808

### CODE REVIEW

**Review by ch1bo — CHANGES_REQUESTED** [review]
Please keep the README.md at the repository top-level. No need to have it in the packages. Consequently it has to be removed from the .cabal to fix the build.

File: servant-exceptions-server/servant-exceptions-server.cabal [hunk 1]
```diff
@@ -3,14 +3,14 @@ version:               0.1.1
...
+license-file:          LICENSE
 author:                Sebastian Nagel
 maintainer:            sebastian.nagel@ncoding.at
 copyright:             2017 Sebastian Nagel
 category:              Web
 build-type:            Simple
-extra-source-files:    ../CHANGELOG.md
-                     , ../README.md
+extra-source-files:    CHANGELOG.md
+                     , README.md
```
Comment (ch1bo) [inline servant-exceptions-server/servant-exceptions-server.cabal:13]: Just keep the single README.md in repository root.

File: servant-exceptions/servant-exceptions.cabal [hunk 2]
```diff
@@ -3,14 +3,14 @@ version:               0.1.1
...
+license-file:          LICENSE
 author:                Sebastian Nagel
 maintainer:            sebastian.nagel@ncoding.at
 copyright:             2017 Sebastian Nagel
 category:              Web
 build-type:            Simple
-extra-source-files:    ../CHANGELOG.md
-                     , ../README.md
+extra-source-files:    CHANGELOG.md
+                     , README.md
```
Comment (ch1bo) [inline servant-exceptions/servant-exceptions.cabal:13]: Just keep the single README.md in repository root and drop it from the package itself.

Comment (hasufell) [reply servant-exceptions/servant-exceptions.cabal:13]: You won't have any top-level README on hackage then...

**Pull request closed without merging**